From a64c0bfa32e0a25eed8338173ba3889045742865 Mon Sep 17 00:00:00 2001
From: Yin Fengwei <fengwei.yin@intel.com>
Date: Thu, 17 Jan 2019 22:09:22 +0800
Subject: [PATCH 1215/1374] vhm: remove the flooding log from vhm

It could be many flooding message like
"VHM: add hugepage with size=..."
printed to kernel log when setup memory for UOS. Changing it to
pr_debug instead of pr_info to avoid old log flush out by this
message in normal case.

Change-Id: I71ce94ac087f4ccc2a1bc87b6a1380292abd1fe7
Reviewed-by: Xu, Anthony <anthony.xu@intel.com>
Reviewed-by: Zhao Yakui <yakui.zhao@intel.com>
Signed-off-by: Yin Fengwei <fengwei.yin@intel.com>
---
 drivers/vhm/vhm_hugetlb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/vhm/vhm_hugetlb.c b/drivers/vhm/vhm_hugetlb.c
index 34ebbd9..72bbef13 100644
--- a/drivers/vhm/vhm_hugetlb.c
+++ b/drivers/vhm/vhm_hugetlb.c
@@ -101,7 +101,7 @@ static int add_guest_map(struct vhm_vm *vm, unsigned long vm0_gpa,
 	if (vm->max_gfn < max_gfn)
 		vm->max_gfn = max_gfn;
 
-	pr_info("VHM: add hugepage with size=0x%lx, vm0_gpa=0x%llx,"
+	pr_debug("VHM: add hugepage with size=0x%lx, vm0_gpa=0x%llx,"
 		" and its guest gpa = 0x%llx, vm max_gfn 0x%x\n",
 		map->size, map->vm0_gpa, map->guest_gpa, vm->max_gfn);
 
-- 
2.7.4

